lib/global/constants.ts

```diff
@@ -0,0 +1,6 @@
+export default {
+    subgraphs: {
+        'mainnet-gauge': 'https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-gauges-beta',
+        'optimism-gauge': 'https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-gauges-optimism-beta',
```

Comment:
can you double check, I think it is now in the "non-beta" subgraph

Reply:
ok